docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -1164,7 +1174,7 @@ For ``TIMESTAMP WITH TIME ZONE`` and ``TIME WITH TIME ZONE`` data types, if time
 be used as Trino value. Otherwise result time zone will be set to ``UTC``.
 
 ``avro`` decoder
-^^^^^^^^^^^^^^^^
+""""""""""""""""
```

Comment:
Could you use same naming between decoder and encoder? 
<img width="245" alt="Screen Shot 2022-10-21 at 8 54 25" src="https://user-images.githubusercontent.com/6237050/197132326-4d9c95a5-5410-436e-841e-4b4d0fd44fb2.png">

Reply:
<img width="257" alt="image" src="https://user-images.githubusercontent.com/3756245/197600241-4ab5fc06-3ae5-454d-a684-168ff3a018ff.png">

Reply:
@ebyhr  thank you for your review